src/en/authors-charm-benchmarks.md

```diff
@@ -1,4 +1,4 @@
-Title: Benchmarking Juju services  
+Title: Benchmarking Juju applications  
```

Comment:
All of these author-* docs are to lesser or greater extents, legacy docs now. They were supposed to be updated by eco but never were. I guess it doesn't hurt to fix up service/application, but we didn't bother in the past as a lot of them are orphaned and effectively unmaintained anyway. I think fixing up the legacy docs is a larger conversation/task